functions/src/logs.ts

```diff
@@ -77,7 +77,19 @@ export function updateDocument(id: string, data: Record<string, any>) {
  */
 export function deleteDocument(id: string) {
   logger.info(
-    `Deleting document ${id} in Meilisearch index ${config.meilisearchIndex}`
+    `Deleting document ${id} in Meilisearch index ${config.meilisearch.indexUid}`
+  )
+}
+
+/**
+ * Log set searchable fields on index in Meilisearch.
+ * @param {string[]} searchableFields Searchable fields.
+ */
+export function updateSearchableFields(searchableFields: string[]) {
+  logger.info(
+    'Update searchable fields',
+    searchableFields,
+    ` in Meilisearch index ${config.meilisearch.indexUid}`
```

Comment:
By doing this, you'r implicitely casting the array `searchableFields` in string which transform the string the following way: 
```
> const a = [1, 2, 3]
> a + ''
'1,2,3'
```
Is this what you want?

Also you can still add `searchableFields` using the `${searchableFields}` inside the backticks

Reply:
I'm sure to understand but this is just a function to print information, it will be printed this ways:
```
    {"severity":"INFO","message":"Update searchable fields [ 'title', 'overview' ]  in Meilisearch index movies"}
```